.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
Since we are no longer using projen, I introduced husky to manage hooks. Husky automatically registers git hooks on `yarn install`.

Reply:
Husky always annoyed me because it doesn't align with how I use squash commits. The commits in a pr are irrelevant for the most part. How do you envision the workflow using husky?

Reply:
Not sure what you mean. All husky is doing right now is installing an equivalent pre-commit hook to the one projen had set up.

Reply:
What does that hook do? I was referring to hooks that validate the commit title.

Reply:
It runs `yarn typecheck` and `yarn lint-staged`. This behavior was already present. This was only included to maintain the same functionality. It shouldn't change the workflow.

Reply:
Husky just runs hooks, you have to add things to the hooks, we just have lint-staged.

Reply:
Got it. I think I didn't understand what husky did - I've only seen it used to validate commit titles. Makes sense that it's running the linters. No problem

Reply:
Husky is fine, do you have it all orchestrated for husky 6? it annoyed me that you need to `init` per user to get husky working after v6.

Reply:
> Husky is fine, do you have it all orchestrated for husky 6? it annoyed me that you need to `init` per user to get husky working after v6.

Interesting, it just worked when I tried it, didn't have me run init. Let me double check the docs.

Reply:
> Husky is fine, do you have it all orchestrated for husky 6? it annoyed me that you need to `init` per user to get husky working after v6.

As far as I can tell, it will auto install, at least with yarn.